shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/FirebaseAuthenticationApi.kt

```diff
@@ -42,7 +39,7 @@ class FirebaseAuthenticationStore : AuthenticationStore {
         return try {
             httpClient.post {
                 url("$BASE_URL$endpoint")
-                parameter("key", apiKey)
+                parameter("key", "") // add api key here
```

Comment:
Would be nice to add a link to the firebase console here

Reply:
Added